src/editor/configuration.js

```diff
@@ -0,0 +1,45 @@
+import Configuration from "../config/configuration"
+import lexxyConfig from "../config/lexxy"
+import { camelize } from "../helpers/string_helper"
+
+export default class EditorConfiguration extends Configuration {
+  static attributes = [ "attachments", "markdown", "single-line", "toolbar" ]
+
+  #editorElement
+
+  constructor(editorElement) {
+    super()
+    this.#editorElement = editorElement
+    this.merge(this.#overrides)
+  }
+
+  get(path) {
+    if (this.#preset) {
+      return super.get(path) ?? lexxyConfig.get(`${this.#preset}.${path}`) ?? lexxyConfig.get(`default.${path}`)
```

Comment:
Why checking the path before prepending the prefix path segment? I think this should be more deterministic. It should invoke the parent with the path prepending the `editorElement.config` segment. 

I am not sure about the inheritance here. `Configuration` feels like a simple data structure you use. I think `EditorConfiguration` could have an attribute configuration that it uses to delegate the `get` and `set`, but inheriting from it feels slightly off to me.